test_runner/regress/test_compatibility.py

```diff
@@ -340,6 +342,17 @@ def check_neon_works(env: NeonEnv, test_output_dir: Path, sql_dump_path: Path, r
         existing_initdb_timeline_id=timeline_id,
     )
 
+    members_sks = [env.safekeepers[0]]
```

Comment:
This is an incorrect patch. The problem with the compat tests was that the snapshot is created without `--timelines-onto-safekeepers`. It should be handled in https://github.com/neondatabase/neon/pull/12203 now

Reply:
removed